index.ts

```diff
@@ -13,21 +14,24 @@ class ConwayError extends Error implements IConwayError {
   readonly rootContext: string;
   readonly contextsChunk: string;
   readonly originalError?: OriginalError;
+  readonly extendedParams?: ExtendedParams;
 
   constructor(
     name: string,
     rootContext: string,
     contextsChunk: string,
     message: string,
     emit: EmitFn,
-    originalError?: OriginalError
+    originalError?: OriginalError,
+    extendedParams?: ExtendedParams
   ) {
     super(message);
     this.name = name;
     this.rootContext = rootContext;
     this.contextsChunk = contextsChunk;
     this.originalError = originalError;
     this.emit = emit;
+    this.extendedParams = extendedParams;
```

Comment:
**sugggestion:** 

maybe not extendedParams but just params? 

Because it is not obvious what we "extend". `Params` is enough clear.